Source/player.cpp

```diff
@@ -3007,7 +2978,6 @@ void CheckNewPath(int pnum, bool pmWillBeCalled)
 				if (player.destAction == ACTION_ATTACKMON || player.destAction == ACTION_ATTACKPLR) {
 
 					if (player.destAction == ACTION_ATTACKMON) {
-						i = player.destParam1;

```

Comment:
3039 nolonger overwrites thease and so we can keep using the value set at line 2965